tests/apps/moderatorfeedback/apps.py

```diff
@@ -3,4 +3,4 @@
 
 class Config(AppConfig):
     name = 'tests.apps.moderatorfeedback'
-    label = 'a4test_moderatorfeedback'
+    label = 'moderatorfeedback'
```

Comment:
Why did you change this?

Reply:
because the migrations would fail otherwise on Django 3.x. Weirdly it now fails on django 2.x